sklearn/ensemble/_stacking.py

```diff
@@ -634,6 +585,27 @@ def _validate_final_estimator(self):
                 )
             )
 
+    def _validate_estimators(self):
+        if len(self.estimators) == 0:
+            raise ValueError(
+                "Invalid 'estimators' attribute, 'estimators' should be a "
+                "non-empty list of (string, estimator) tuples."
+            )
+        names, estimators = zip(*self.estimators)
+        # defined by MetaEstimatorMixin
```

Comment:
I can see that this is a copy and paste from the original `_BaseHeterogeneousEnsemble._validate_estimators`. I prefer not to include them because these comments can become out of date if we refactor `_validate_names` out of `MetaEstimatorMixin` in the future.

```suggestion
```

Side note: I think most IDE's can jump to a function definition pretty easily now.